localisation/00_wc_vanilla_replacements.csv

```diff
@@ -292,4 +292,19 @@ EVTDESC_MNM_20081_CHILD;My fellows[Root.GetInSociety] have managed to insert the
 EVTDESC_MNM_20083;My competent brethren[Root.GetInSociety] managed to snatch [From.GetTitledFirstNameNoRegnal] from [From.GetPossPronoun] guards and bring [From.GetObjectPronoun] to me in a sack!;;;;;;;;;;;;;x
 EVTDESC_MNM_20084;My incompetent brethren[Root.GetInSociety] failed to capture [From.GetTitledFirstNameNoRegnal], though apparently they slew a guard and a servant in the attempt. Even worse, it appears that [From.GetFirstName] has started to suspect my involvement in the ambush.;;;;;;;;;;;;x;
 stop_abduct_decision_desc;Call off my [Root.GetSocietyMembers] from abducting my current target.;;;;;;;;;;;;;x
-bloodthirsty_mass_sacrifice_decision;Hold Blood Rituals;;;;;;;;;;;;;x
\ No newline at end of file
+bloodthirsty_mass_sacrifice_decision;Hold Blood Rituals;;;;;;;;;;;;;xEVTDESC7880;[Root.PlotTarget.GetTitledName] will soon travel in a convoy on an isolated road. Perhaps if we gathered some mercenaries and disguised them as [Root.PlotTarget.GetHighwaymenType]... crime is on the rise, after all.;;;;;;;;;;;;;x
```

Comment:
Broken merge?

Reply:
Fixed in 32d056c13669eb6a19d1b03951787f589f46499b.